crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -672,7 +670,7 @@ mod test {
         );
         // signal server exit
         tx.send(CloseReason::Interrupt).unwrap();
-        handle.await.unwrap();
+        handle.await.unwrap().unwrap();
```

Comment:
The server can't actually return an error here. It only does so if package discovery fails and our mock one is infallible. This is mainly just for the lint but if it were to panic here it would imply an issue with the logic of the test

Reply:
The server can't actually return an error here. It only does so if package discovery fails and our mock one is infallible. This is mainly just for the lint but if it were to panic here it would imply an issue with the logic of the test